### PR TITLE
Replaces singularity in CLI examples 

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
Signed-off-by: Pedro Alves Batista <pedro.pesquisapb@gmail.com>

Replaces `singularity` to `{command}` in CLI examples.

* Fixes:
#13

### CODE REVIEW

File: admin_quickstart.rst [hunk 1]
```diff
@@ -194,16 +194,16 @@ Download {Project} from a GitHub release
...
 release to install, you can run the following commands to proceed with
 the installation.
 
 .. code::
 
     $ export VERSION={InstallationVersion} && # adjust this as necessary \
-        wget https://github.com/hpcng/singularity/releases/download/v${VERSION}/singularity-${VERSION}.tar.gz && \
-        tar -xzf singularity-${VERSION}.tar.gz && \
-        cd singularity
+        wget https://github.com/hpcng/{command}/releases/download/v${VERSION}/{command}-${VERSION}.tar.gz && \
```
Comment (DrDaveD) [inline admin_quickstart.rst:204]: This github URL should use `{orgrepo}` instead (in another PR)

File: appendix.rst [hunk 2]
```diff
@@ -11,58 +11,58 @@ ownership and permissions required to use the `setuid` workflow:
...
-   var/singularity/mnt/session root:root 755 (drwxr-xr-x)
+   var/{command} root:root 755 (drwxr-xr-x)
+   var/{command}/mnt root:root 755 (drwxr-xr-x)
+   var/{command}/mnt/session root:root 755 (drwxr-xr-x)
 
    # Main executables
-   bin/singularity root:root 755 (-rwxr-xr-x)
-   bin/run-singularity root:root 755 (-rwxr-xr-x)
+   bin/{command} root:root 755 (-rwxr-xr-x)
+   bin/run-{command} root:root 755 (-rwxr-xr-x)
```
Comment (DrDaveD) [inline appendix.rst:20]: This one has to stay as `run-singularity`

File: conf.py [hunk 3]
```diff
@@ -121,7 +121,7 @@
 html_context = {
     'display_github': True,
     'github_user': 'hpcng',
-    'github_repo': 'singularity-admindocs',
+    'github_repo': '{command}-admindocs',
```
Comment (DrDaveD) [inline conf.py:124]: Make this apptainer, I found that conf.py does not support replacements.

Comment (DrDaveD) [reply conf.py:124]: Likewise several other cases in this file below.

File: conf.py [hunk 4]
```diff
@@ -279,7 +279,7 @@
 
 # The unique identifier of the text. This can be a ISBN number
 # or the project homepage.
-epub_identifier = 'https://github.com/hpcng/singularity-admindocs'
+epub_identifier = 'https://github.com/hpcng/{command}-admindocs'
```
Comment (DrDaveD) [inline conf.py:282]: Leave this for the PR on github URL

File: conf.py [hunk 5]
```diff
@@ -299,4 +299,4 @@
 from replacements import *
 
 # lexer for Singularity definition files (added here until it is upstreamed into Pygments).
-lexers['singularity'] = SingularityLexer(startinline=True)
+lexers['{command}'] = SingularityLexer(startinline=True)
```
Comment (DrDaveD) [inline conf.py:302]: In order to convert the lexer a lot more has to be done as in the second commit of apptainer/apptainer-userdocs#54.  However, I do not see anything of the form
```
.. code:: {command}
```
so I think the lexer isn't actually used and can be deleted.

File: installation.rst [hunk 6]
```diff
@@ -380,16 +380,16 @@ Download {Project} from a release
 
 You can download {Project} from one of the releases. To see a full
 list, visit `the GitHub release page
-<https://github.com/hpcng/singularity/releases>`_. After deciding on a
+<https://github.com/hpcng/{command}/releases>`_. After deciding on a
```
Comment (DrDaveD) [inline installation.rst:383]: Don't change github URL in this PR

File: installation.rst [hunk 7]
```diff
@@ -620,12 +620,12 @@ directories to completely remove {Project}.
...
-       /usr/local/var/singularity \
-       /usr/local/etc/singularity \
-       /usr/local/bin/singularity \
-       /usr/local/bin/run-singularity \
-       /usr/local/etc/bash_completion.d/singularity
+       /usr/local/libexec/{command} \
+       /usr/local/var/{command} \
+       /usr/local/etc/{command} \
+       /usr/local/bin/{command} \
+       /usr/local/bin/run-{command} \
```
Comment (DrDaveD) [inline installation.rst:627]: this stays at run-singularity

File: installation.rst [hunk 8]
```diff
@@ -788,7 +788,7 @@ Then issue the following commands to bring up the Virtual Machine.
 
 .. code::
 
-   $ export VM=sylabs/singularity-3.8-ubuntu-bionic64 && \
+   $ export VM=sylabs/{command}-3.8-ubuntu-bionic64 && \
```
Comment (DrDaveD) [inline installation.rst:791]: There was no apptainer-3.8 so this might as well stay at sylabs/singularity for now.

File: installation.rst [hunk 9]
```diff
@@ -380,16 +380,16 @@ Download {Project} from a release
...
 list, visit `the GitHub release page
-<https://github.com/hpcng/singularity/releases>`_. After deciding on a
+<https://github.com/hpcng/{command}/releases>`_. After deciding on a
 release to install, you can run the following commands to proceed with
 the installation.
 
 .. code::
 
    $ export VERSION={InstallationVersion} && # adjust this as necessary \
        wget https://github.com/hpcng/singularity/releases/download/v${VERSION}/singularity-${VERSION}.tar.gz && \
```
Comment (DrDaveD) [inline installation.rst:390]: This one should have `{command}` as part of the file name at the end instead of singularity

File: conf.py (hunk 5 above)
Comment (pedroalvesbatista) [reply conf.py:302]: Delete this whole part regarding lexer and wipe out any reference to it (like this `.. code::` sections) ?

Comment (DrDaveD) [reply conf.py]: Delete references to the lexer and pygments here in conf.py and remove pygments_singularity.py.  The `.. code::` sections that are there stay the same; the point is that none of them refer to a lexer called `{command}` like many of the code sections did in userdocs.

File: conf.py [hunk 10]
```diff
@@ -279,7 +279,7 @@
 
 # The unique identifier of the text. This can be a ISBN number
 # or the project homepage.
-epub_identifier = 'https://github.com/hpcng/singularity-admindocs'
+epub_identifier = 'https://github.com/hpcng/apptainer-admindocs'
```
Comment (DrDaveD) [inline conf.py:282]: There is no such thing as hpcng/apptainer.  Anyway this will be done in a different PR.

File: configfiles.rst [hunk 11]
```diff
@@ -459,8 +459,13 @@ option:
 
 .. code::
 
+<<<<<<< HEAD
    $ sudo singularity config global --dry-run --set "bind path" /etc/resolv.conf
    # {ENVPREFIX}.CONF
+=======
+   $ sudo {command} config global --dry-run --set "bind path" /etc/resolv.conf
+   # SINGULARITY.CONF
+>>>>>>> Replaces singularity to {command} in CLI examples #13
```
Comment (DrDaveD) [inline configfiles.rst:468]: Fix unresolved merge.

**Review by DrDaveD — CHANGES_REQUESTED** [review]
There are conflicts to resolve too

File: admin_quickstart.rst [hunk 12]
```diff
@@ -194,16 +194,20 @@ Download {Project} from a GitHub release
 
 You can download {Project} from one of the releases. To see a full
 list, visit `the GitHub release page
+<<<<<<< HEAD
 <https://github.com/{orgrepo}/releases>`_.  After deciding on a
+=======
+<https://github.com/hpcng/{command}/releases>`_.  After deciding on a
+>>>>>>> 162907d... Replaces singularity to {command} in CLI examples #13
```
Comment (ikaneshiro) [inline admin_quickstart.rst:201]: This merge conflict needs to be resolved

**Review by ikaneshiro — APPROVED** [review]

File: conf.py (hunk 5 above)
Comment (pedroalvesbatista) [reply conf.py]: Then the `.. code::` sections can be backed to `apptainer` instead of `{command}` ?

**Review by DrDaveD — APPROVED** [review]

File: conf.py (hunk 5 above)
Comment (DrDaveD) [reply conf.py]: The point is that there aren't any in admindocs like there were in userdocs.

**Pull request merged**